src/test/resources/eager/reverts-simple.expected.jinja

```diff
@@ -1,9 +1,9 @@
 {% for __ignored__ in [0] %}
   {% set my_list = [0, 1] %}{% if deferred %}
-    {% set my_list = [0, 1, 2] %}
+    {% do my_list.append(2) %}
```

Comment:
There are many similar changes like this in the first phase outputs. But they result in the same final output so they're good